src/fty_nut_configurator_server.cc

```diff
@@ -289,6 +289,7 @@ fty_nut_configurator_server (zsock_t *pipe, void *args)
                 if (state_writer.getState().updateFromProto(proto))
                     state_writer.commit();
                 agent.onUpdate();
+                fty_proto_destroy (&proto);
```

Comment:
Just in case: are you certain there are no consumers of this routine which actually expect the proto instance being not freed in it?

Reply:
considering the variable scope of proto, no problem